src/app/pets/pets.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="row">
+    <div class="col-xs-4">
+        <H1>Pet List: {{owner.lastName}} {{owner.firstName}}</H1>
+    </div>
+    <div class="col-xs-1" style="text-align: center; margin-top: 20px;">
+        <button class="btn btn-success" (click)="onNewPet()" >Add</button>
+    </div>
+</div>
+<hr>
+<div class="row">
+    <div class="col-md-5">
+        <app-pet-list></app-pet-list>
+    </div>
+    <div class="col-md-7">
+        <router-outlet></router-outlet>
```

Comment:
<router-outlet></router-outlet>
no need for this ...if its needed then something else is wrong